controller.go

```diff
@@ -36,6 +36,17 @@ type controller interface {
 	getMeasuredServices() map[string]measuredService
 }
 
+//Both enriched structs are used for applying the services' system code to the ft check results
+type enrichedHealthResult struct {
+	HealthResult fthealth.HealthResult
+	Checks       []enrichedCheckResult
+}
+
+type enrichedCheckResult struct {
+	CheckResult fthealth.CheckResult
+	SystemCode  string `json:"systemCode"`
+}
+
```

Comment:
**suggestion (non-blocking):** I think that if we embed `fthealth.HealthResult` and `fthealth.CheckResults` the changes throughout the code will be less.